### PR TITLE
[T2A2][W15-A2] Liu Ziyang

### PR DESCRIPTION
**Pull request by VeryLazyBoy**

### CODE REVIEW

File: src/seedu/addressbook/AddressBook.java [hunk 1]
```diff
@@ -794,11 +795,11 @@ private static void addPersonToAddressBook(String[] person) {
      * @return true if the given person was found and deleted in the model
      */
     private static boolean deletePersonFromAddressBook(String[] exactPerson) {
-        final boolean changed = ALL_PERSONS.remove(exactPerson);
-        if (changed) {
+        final boolean hasChanged = ALL_PERSONS.remove(exactPerson);
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:798]: good job spotting this naming violation. makes it easier to understand that this variable is a boolean/flag.

File: src/seedu/addressbook/AddressBook.java [hunk 2]
```diff
@@ -1150,8 +1151,9 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String s, String prefix) {
+        int indexOfDataWithoutPrefix = prefix.length();
+        return s.substring(indexOfDataWithoutPrefix);
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:1156]: are you sure all strings passed in contain the prefix?

File: src/seedu/addressbook/AddressBook.java [hunk 3]
```diff
@@ -1150,8 +1151,9 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:1151]: method comments must be also updated if there are changes to behaviour or method signature

File: src/seedu/addressbook/AddressBook.java [hunk 4]
```diff
@@ -1150,8 +1151,9 @@ private static String getUsageInfoForExitCommand() {
...
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String s, String prefix) {
```
Comment (mauris) [inline src/seedu/addressbook/AddressBook.java:1154]: `s` is not a meaningful variable name.

**Comment by mauris** [comment]
Some comments added. Feel free to add more comment if you have any questions. Once you are done, you may close the PR after reading comments.

File: src/seedu/addressbook/AddressBook.java (hunk 2 above)
Comment (VeryLazyBoy) [reply src/seedu/addressbook/AddressBook.java:1156]: Yes, because the method isPersonDataExtractableFrom has checked the validity.

**Pull request closed without merging**